help/templates/template.cshtml

```diff
@@ -171,9 +171,8 @@
 									<section>
 										<h2>Important Links</h2>
 										<ul class="divided">
-											<li><a href="@(prefix)404.html">Quickstart</a></li>
-											<li><a href="@(prefix)404.html">Installation</a></li>
-											<li><a href="@(prefix)404.html">Modules</a></li>
+											<li><a href="https://fake.build/fake-gettingstarted.html">Getting started</a></li>
+											<li><a href="https://fake.build/fake-fake5-modules.html">Modules</a></li>
```

Comment:
We need to use `@(prefix)` here, otherwise it doesn't work locally or on on other urls.

No worries I'll update it